README.md

```diff
@@ -101,15 +101,21 @@ Citing
 
 If you use pvlib-python in a published work, please cite:
 
-  William F. Holmgren, Clifford W. Hansen, and Mark A. Mikofski.
-  "pvlib python: a python package for modeling solar energy systems."
-  Journal of Open Source Software, 3(29), 884, (2018).
-  https://doi.org/10.21105/joss.00884
```

Comment:
Could we keep the 2018 citation too?  Maybe a category structure like this, which leaves room for future expansion:

```
If you use pvlib-python in a published work, please cite as appropriate:

Project as a whole:
- 2023 JOSS citation
- 2018 JOSS citation

Data retrieval using `pvlib.iotools`:
- 2023 iotools citation
```